start-backend.sh

```
@@ -0,0 +1,43 @@
+
+# Kyros Dashboard - Backend Quick Start Script
+echo "🚀 Starting Kyros Dashboard Backend..."
+
+# Check if we're in the right directory
+if [ ! -f "api/main.py" ]; then
+    echo "❌ Error: Please run this script from the project root directory"
+    exit 1
+fi
+
+# Navigate to API directory
+cd api
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)